TypeCobol.Test/Parser/Programs/Cobol85/ExecSql/ExecSqlWithFromClause.rdz.cbl

```diff
@@ -0,0 +1,14 @@
+       IDENTIFICATION DIVISION.
+       PROGRAM-ID. DVZZMFT3.
+       DATA DIVISION.
+       WORKING-STORAGE SECTION.
+         EXEC SQL 
```

Comment:
All EXEC SQL must start at column 12, so check your test files and remember it for future tests